run-tests.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+function setup() {
+    cd ./lib/validator && npm install
+}
+
+SUBCOMMAND=$1
+
+# set rust validator worker env var
+export RUST_VALIDATOR_WORKER=./target/debug/validator_worker 
+
+# RUN Tests
+if [ "$SUBCOMMAND" == "--setup" ]; then
+    echo "Setup validator tests"
+    setup    
+else
+    cargo build -p validator_worker
```

Comment:
NOTE: this might not play well with CI because of the `target` it's building for.